packages/terra-navigation-side-menu/src/NavigationSideMenu.module.scss

```diff
@@ -84,4 +84,15 @@
     width: 100%;
     word-wrap: break-word; /* For IE 10 and IE 11 */
   }
+
+  .drill-in-title {
```

Comment:
Can we keep the common styles between title and drill-in-title in .title and conditionally add-in the differences in .drill-in-title

Reply:
Updated. Thanks